webxr-api/registry.rs

```diff
@@ -185,10 +187,10 @@ impl MainThreadRegistry {
     }
 
     fn request_session(&mut self, mode: SessionMode) -> Result<Session, Error> {
-        let webgl = self.webgl.as_ref().ok_or(Error::NoMatchingDevice)?;
+        let swap_chains = self.swap_chains.as_mut().ok_or(Error::NoMatchingDevice)?;
```

Comment:
maybe a new error variant?

Reply:
Well, I think the spec says the only error this can return is NoMatchingDevice?

Reply:
Sure, but we can be more specific in our API if we want. r=me either way.